src/sass/abstracts/_variables.scss

```diff
@@ -25,6 +25,7 @@ $fs-11: 2.25rem; // 36px
 $fs-12: 2.5rem; // 40px
 $fs-14: 3rem;
 $fs-16: 3.5rem; // 56px
+$fs-18: 1.625rem; // 26px
```

Comment:
would be cool to keep these values incremental.
fs-6 body text mobile
fs-8 body text desktop
fs-10 small headline
fs-12 

etc..